examples/structs/structs.go

```diff
@@ -38,4 +48,7 @@ func main() {
 	// Structs are mutable.
 	sp.age = 51
 	fmt.Println(sp.age)
+
+	// Call our constructor
```

Comment:
I'd move this up after the `// An `&` prefix yields a pointer to the struct.` example, because it does exactly the same but with a function. The comment could say something like:

    // It's idiomatic to encapsulate new struct creation in constructor functions